Assets/Scripts/Core/CameraBillboard.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.Design.Serialization;
+using UnityEngine;
+
+public class CameraBillboard : MonoBehaviour
+{
+    private Camera _cam;
+
+    public bool enableBillboard = false; 
+    
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        _cam = Camera.main;
+    }
+
+    // Update is called once per frame
+    void LateUpdate()
```

Comment:
Lägg till private + Jag tycker att du borde slänga in logiken som är i Start och LateUpdate till separata metoder, för då får du ett tydligt namn på vad som händer + att det blir lättare att testa sen när man gör unittests tror jag (killgissar, men vi gör så på jobbet)